docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -207,7 +229,9 @@ For more information, see xref:security-oauth2.adoc[Using OAuth2].
 
 == Choosing between OpenID Connect, SmallRye JWT, and OAuth2 authentication mechanisms
 
-Use the following information to help you to decide which authentication mechanism to use to secure your Quarkus applications:
+Use the following information to select the proper token authentication mechanism for your applications.
```

Comment:
Style suggestion: I think the use of "proper" here could be a bit ambiguous as in, would a user know which is proper mechanism for their application and which is now? I think "appropriate" might be better.
Nitpick: For maximum clarity, I think I would keep the previous wording for the end of the sentence, that is,  "Use the following information to select the appropriate token authentication mechanism to secure your Quarkus applications."

Reply:
OK, will apply.